packages/block-editor/src/components/block-list/subdirectory-icon.js

```diff
@@ -0,0 +1,21 @@
+/**
+ * WordPress dependencies
+ */
+import { SVG, Path } from '@wordpress/components';
+
+const Subdirectory = ( { ...extraProps } ) => (
+	<SVG
+		aria-hidden
+		role="img"
+		focusable="false"
```

Comment:
Are these props actually used? It feels like this file should only have the icon and let the parent specify these adjustments, but if this is only used on native, maybe we don't need them at all?

Reply:
I just copied it from the dashicon component. I think they can be removed and passed by extra props if needed.